op-e2e/actions/upgrades/helpers/config.go

```diff
@@ -8,7 +8,6 @@ import (
 // ApplyDeltaTimeOffset adjusts fork configuration to not conflict with the delta overrides
 func ApplyDeltaTimeOffset(dp *e2eutils.DeployParams, deltaTimeOffset *hexutil.Uint64) {
```

Comment:
Uses of this function will be replaced by the more generic `ActivateForkAtOffset` in a follow-up.